uni/lib/l10n/intl_pt_PT.arb

```diff
@@ -294,6 +294,8 @@
   "@terms": {},
   "title": "Título",
   "@title": {},
+  "uc_info": "Abrir página da UC na app",
```

Comment:
Probably just "Abrir página da UC" is fine. Wdyt @bdmendes?

Reply:
Agreed